### ISSUE TITLE
HW 2 - COMPLETE

### ISSUE THREAD

**Issue by HannahCBrunelle** [HannahCBrunelle/Brunelle_SpatialEcology#4]
@fitzLab-AL and @Emlbcohen

**Comment by fitzLab-AL** [comment]
@HannahCBrunelle Have a look at the comments and edits I made to your HW2 script. There were a few places where the code failed due to errors. I tried to fix these so the code was run, but I tended to get lost. I would highly recommend organizing your code into discrete chunks. I found it really hard to follow your code and to debug errors. Also, please see the solution I posted - there were a few instances where you deviated a bit from the assignment.  Let me know if you would like to discuss or if you have any questions.

GRADE: 7.5/10
